README.md

```diff
@@ -1,33 +1,33 @@
-ShakaCode's JavaScript Style Guide (Also see our Ruby one: https://github.com/shakacode/style-guide-ruby)
+ShakaCode's JavaScript style guide. (You should also check out [our Ruby style guide](https://github.com/shakacode/style-guide-ruby))
 
-See [eslint-config-shakacode](./eslint-config-shakacode/README.md) for an automated way to follow our guidelines.
+See [eslint-config-shakacode](./eslint-config-shakacode/README.md) for setting up your linter to follow these style guidelines.
 
 # ShakaCode JavaScript Style Guide
-
-Our **JavaScript Style Guide** is relatively simpler because we're simply using the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) with it's associated `.eslintrc` and `.jscsrc` files.
+Our **JavaScript Style Guide** is relatively simple because we're leveraging the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) and its associated `.eslintrc` and `.jscsrc` files.
 
 Here's a few notes on top of that:
 
-1. **Line Length**: We're sticking with the AirBnb Standard of 120, and any strings greater than 100 should be broken up to multiple lines. Airbnb moved to 120, here's [why](https://github.com/airbnb/javascript/pull/458).
-2. Definitely follow the [AirBnb React Guide](https://github.com/airbnb/javascript/blob/master/react/README.md)
-3. Use ES6 classes for React.
+1. **Line Length**: We're sticking with the AirBnb standard of 120. See [their reasoning](https://github.com/airbnb/javascript/pull/458). Additionally, any strings greater than 100 should be broken up into multiple lines. 
+1. Check out the [AirBnb React Guide](https://github.com/airbnb/javascript/blob/master/react/README.md) keeping in mind the exceptions listed below.
+1. Use ES6 classes for React components.
 
 # Exceptions From AirBnb JavaScript Style Guide (React Guide only)
-
 ## Use the Official Docs from Facebook
 We mostly folow the official [FB docs on ES6 for React](https://facebook.github.io/react/docs/reusable-components.html#es6-classes). We initialize state in the constructor. 
 
 Notable differences:
 
 1. ES7 static syntax for the propTypes and defaultProps
-2. Use lodash `_.bindAll` for callback bindings.
+1. Use lodash `_.bindAll` for callback bindings.
 
-For example:
+An example React component:
 
-```javascript
+```js
```

Comment:
sorry for noise, was trying something and thought I changed it back, either way works tho